modules/nextflow/src/main/groovy/nextflow/config/ConfigBuilder.groovy

```diff
@@ -643,7 +646,7 @@ class ConfigBuilder {
             if( cmdRun.withDag != '-' )
                 config.dag.file = cmdRun.withDag
             else if( !config.dag.file )
-                config.dag.file = GraphObserver.DEF_FILE_NAME
+                config.dag.file = 'dag.dot'
```

Comment:
why this name does use the timestamp?

Reply:
It didn't make sense to me for the DAG to be timestamped, because it doesn't change very often.

Reply:
On second thought maybe the DAG should be timestamped, because the DAG can look different from run to run based on piipeline params that affect the DAG structure (eg. enabling/disabling steps).

Reply:
> because the DAG can look different from run to run based on pipeline params that affect the DAG structure (eg. enabling/disabling steps).

Agreed, this would make for a good pipeline dev-time experience.